### PR TITLE
[940] Initial project framework

### PR DESCRIPTION
**Pull request by miklosszegedi**
Author: Miklos Szegedi

### CODE REVIEW

**Comment by miklosszegedi** [comment]
Merging per approval of @alavallee21

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Flow-go
+Copyright 2019-2020 Dapper Labs, Inc.
+
+This product includes software developed at Dapper Labs, Inc. (https://www.dapperlabs.com/).
```
Comment (koko1123) [inline NOTICE:4]: we don't need this file and the `.md` files below, we just need the `README.md` for now

File: src/main/api-service.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Printf("Hello World!\n")
+}
```
Comment (koko1123) [inline src/main/api-service.go:7]: Let's start with at least an enumeration of all the included functions in the API

Comment (miklosszegedi) [reply src/main/api-service.go:7]: Thanks. We have a newer pull request now. I will update it later today.

File: NOTICE (hunk 1 above)
Comment (miklosszegedi) [reply NOTICE:4]: All right.

**Pull request merged**